cmd/osbuild-image-tests/main_test.go

```diff
@@ -169,10 +180,10 @@ func trySSHOnce(ns netNS) error {
 // testSSH tests the running image using ssh.
 // It tries 20 attempts before giving up. If a major error occurs, it might
 // return earlier.
-func testSSH(t *testing.T, ns netNS) {
+func testSSH(t *testing.T, address string, privateKey string, ns *netNS) {
 	const attempts = 20
 	for i := 0; i < attempts; i++ {
-		err := trySSHOnce(ns)
+		err := trySSHOnce(address, privateKey, ns)
 		if err == nil {
```

Comment:
you can use require.NoError here as well

Reply:
I want to pass the test immediately when `err == nil`, therefore `require.NoError` doesn't make much sense here.